test/e2e/new-link-behavior/typescript/pages/index.tsx

```diff
@@ -8,7 +8,7 @@ import {
 import NextLink, { LinkProps } from 'next/link'
 
 type NativeButtonProps = Omit<
-  HTMLProps<HTMLButtonElement>,
+  HTMLProps<HTMLAnchorElement>,
```

Comment:
With new link behavior, the `<Link />` will always render a `HTMLAnchorElement`.